static/stripe.html

```diff
@@ -326,12 +329,12 @@ <h4 class="col s12">Thank you for your donation!</h4>
         // Display some info
         $('#payment-form .amount').text('$' + (Math.floor(data.amount) / 100).toFixed(2))
         $('#payment-form .email').text(data.email)
-        if (data.amount < 500) {
-            $('#payment-form .show-if-not-premium').removeClass('hidden')
-            $('#payment-form .show-if-premium').addClass('hidden')
-        } else {
+        if (data['premium'] === true) {
```

Comment:
This change means clientside doesn't "know" what amount triggers going premium - so if we changed the amount serverside we wouldn't have to worry about the wrong amount being cached clientside

Reply:
Unrelatedly `data` here and also `currentPayment` could really do with better names - they are the data returned from Impact API's `/stripe/createpayment` - so they are related to PaymentIntents but aren't the same thing